### DISCUSSION TITLE
Google drive link is bad, please consider better hosting

### DISCUSSION THREAD

**Discussion by unicornsandcandies** [YoungFellow-le/photoshop-22-linux#20] [Ideas]
- Downloading Photoshop files and compnents if not already downloaded...

Access denied with the following error:

        Cannot retrieve the public link of the file. You may need to change
        the permission to 'Anyone with the link', or have had many accesses. 

You may still be able to access the file from the browser:

         https://drive.google.com/uc?id=1VqIUUzCDuyxOXM-q99ySKEKU_n8DbB26

**Comment by albakhtari** [comment]
The file has been accessed too many times, I'm uploading a fresh copy now.

**Comment by albakhtari** [comment]
This should solve the issue.
Better hosting is out of my budget.

**Comment by maplepy** [comment]
Still had the issue, had to download manually

**Comment by albakhtari** [comment]
Have you updated the local repository?
Run the following in the folder in which you cloned photoshop:
```bash
cd path/to/photoshop-22-linux
git pull
```

**Edit:** I just tested the link, and it is working just fine.

**Comment by maplepy** [comment]
yep, i just discovered the repo yesterday so it was up to date already

**Comment by albakhtari** [comment]
I just added a note in the README.md file, do check that out.
Also, can I ask how you found out about my repo?

**Comment by maplepy** [comment]
I guess I did something wrong, which is totally on me, I'll check that later tonight too.

Sure! Basically I was looking to download PS on Linux, I remembered seeing a photoshop CC running on a linux some time ago, the one with the rick and morty splash screen. I used it for some time but it was kinda buggy for me at the time. I needed a strong image editor and wanted to download that one again but instead I found this repo and another, Milli's one and I was interested in your install script so I checked it out and tried it and it was great! Thank you

**Comment by albakhtari** [comment]
You're most welcome!
I also started out using that same Photoshop repo that you mentioned, but it was as you described.
Then I found @MiMillieuh's repo, I forked their repo... Then [decided to migrate](https://github.com/YoungFellow-le/Linux-PhotoshopCC-v22/discussions/8) to my own repo.

> I guess I did something wrong, which is totally on me, I'll check that later tonight too.

Don't worry about it.

**Comment by maplepy** [comment]
Thank you very much!

By the way I'm making some improvements to your script, so it's a bit easier to change and read, would you want me to make a pull request for it or should I just fork it ?

As of the google drive issue, it was because I was missing gdown I think so that was on me aha

**Comment by albakhtari** [comment]
Sorry for the late reply again.

> By the way I'm making some improvements to your script, so it's a bit easier to change and read, would you want me to make a pull request for it or should I just fork it ?

Please do, I will be very happy to merge your contributions into the main branch



> As of the google drive issue, it was because I was missing gdown

Didn't the script notify you about that?

**Reply by maplepy** [reply]
Alright :) I'll finish the rework of the script and will send a PR then

I honestly don't remember sorry about that :/ Had a hell of a week aha
